### PR TITLE
Packages/py colorspacious

### PR DESCRIPTION
**Pull request by Bidibulke**
Adding recipe for `colorspacious` (a python package)

### CODE REVIEW

File: lib/spack/docs/basic_usage.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+.. Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline lib/spack/docs/basic_usage.rst]: Please revert this change

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-colorspacious/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    """`colorlog.ColoredFormatter` is a formatter for use with Python's
+    `logging` module that outputs records using terminal colors.
+     """
+
+    homepage = "https://github.com/borntyping/python-colorlog"
+    url      = "https://pypi.io/packages/source/c/colorspacious/colorspacious-1.1.2.tar.gz"
+
+    version('1.1.2', sha256='5e9072e8cdca889dac445c35c9362a22ccf758e97b00b79ff0d5a7ba3e11b618')
+
+    depends_on('py-numpy')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-colorspacious/package.py]: Can you double check dependencies? `py-numpy` doesn't seem to be listed in `setup.py`.

**Review by alalazo — CHANGES_REQUESTED** [review]
Overall LGTM, just a request to double check dependencies.

File: var/spack/repos/builtin/packages/py-colorspacious/package.py (hunk 2 above)
Comment (Bidibulke) [reply var/spack/repos/builtin/packages/py-colorspacious/package.py]: I checked the master branch and the 1.1.2 version and `numpy` is in the *install_requires* : [current_setup](https://github.com/njsmith/colorspacious/blob/58948923b706879a54071568c7501be3f108797c/setup.py#L35).

Comment (alalazo) [reply var/spack/repos/builtin/packages/py-colorspacious/package.py]: @Bidibulke I think I got the issue. I check the [homepage](https://github.com/borntyping/python-colorlog) currently in the package and it doesn't match the source being retrieved. Can you fix that?

Comment (Bidibulke) [reply var/spack/repos/builtin/packages/py-colorspacious/package.py]: The link in your previous comment is for `colorlog`, not `colorspacious`.

File: var/spack/repos/builtin/packages/py-colorspacious/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+from spack import *
+
+
+class PyColorspacious(PythonPackage):
+    """`colorlog.ColoredFormatter` is a formatter for use with Python's
+    `logging` module that outputs records using terminal colors.
+     """
+
+    homepage = "https://github.com/borntyping/python-colorlog"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-colorspacious/package.py]: @Bidibulke The link in my previous comment comes from this package :slightly_smiling_face:

Comment (Bidibulke) [reply var/spack/repos/builtin/packages/py-colorspacious/package.py]: Oups. I didn't see that.

**Comment by scheibelp** [comment]
This still appears to have the wrong homepage

File: var/spack/repos/builtin/packages/py-colorspacious/package.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+from spack import *
+
+
+class PyColorspacious(PythonPackage):
+    """A powerful, accurate, and easy-to-use Python library for doing colorspace
+    conversions.
+    """
+
+    homepage = "https://github.com/borntyping/python-colorlog"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-colorspacious/package.py]:
```suggestion
    homepage = "https://github.com/njsmith/colorspacious"
```

File: var/spack/repos/builtin/packages/py-colorspacious/package.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    url      = "https://pypi.io/packages/source/c/colorspacious/colorspacious-1.1.2.tar.gz"
+
+    version('1.1.2', sha256='5e9072e8cdca889dac445c35c9362a22ccf758e97b00b79ff0d5a7ba3e11b618')
+
+    depends_on('py-numpy')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-colorspacious/package.py]:
```suggestion
    depends_on('py-setuptools', type='build')
    depends_on('py-numpy', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by alalazo — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]
